pallets/manta-pay/README.md

```diff
@@ -0,0 +1,37 @@
+# pallet-manta-pay
+
+This is a pallet that enables MantaPay, a multi-asset, decentralized shielded payment protocol.
+The best way to use this repo is to invoke it with a `manta-runtime`,
+available from [manta](https://github.com/Manta-Network/manta).
+
+__Disclaimer__: The code has not been properly reviewed or audited and is likely to have 
+severe bugs or security pitfalls.Use at your own risk!
+
+## Documentations
+``` sh
+cargo doc --open
+```
+
+## Update pre-computed tokens
+``` sh
+cargo run --release --bin precompute_coins --features=precompute-coins > ./src/benchmark/precomputed_coins.rs
```

Comment:
These are failing with :
error: there is no argument named ORGANIZATION

Reply:
will fix that.